lib/rubocop/cop/lint/uri_regexp.rb

```diff
@@ -43,10 +43,8 @@ def on_send(node)
 
         def autocorrect(node)
           lambda do |corrector|
-            if (captured_values = uri_regexp_with_argument?(node))
-            else
-              captured_values = uri_regexp_without_argument?(node)
-            end
+            captured_values =
```

Comment:
I'm always happy when a new cop catches weird code within our own source code. Really helps to confirm that the cop is useful.